FirebaseCore/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
 - [removed] **Breaking change**: Removed the following unused API.
   - `Options.androidClientID`
   - `Options.trackingID`
+- [removed] The deprecated Vertex AI in Firebase SDK (`FirebaseVertexAI`) has
+  been removed. It has been replaced by the Firebase AI Logic
+  SDK (`FirebaseAI`) to
+  accommodate the evolving set of supported features and services.
+  To start using the new SDK, import the `FirebaseAI` module and use the
+  top-level `FirebaseAI` class. See details in the [migration guide
+  ](https://firebase.google.com/docs/vertex-ai/migrate-to-latest-sdk).
```

Comment:
super nit (don't worry about this unless you need to make another commit for another reason since it auto-redirects):
```suggestion
  ](https://firebase.google.com/docs/ai-logic/migrate-to-latest-sdk).
```